h/static/scripts/controllers/character-limit-controller.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+function CharacterLimitController(element) {
+  var counter = element.parentElement.querySelector(
+    '.js-character-limit-counter');
```

Comment:
Actually I'd rather use a different mechanism or namespace to indicate controllers vs. classes that are there to provide handles for controllers to get references to elements.

I'm keeping an eye out for existing libraries or at least conventions that are a good fit for our circumstances, since I don't want to reinvent wheels if there are good round ones going.